styles/colors-copper.less

```diff
@@ -270,7 +270,3 @@
 @agate-togglebutton-bg-color: @agate-button-bg-color;
 @agate-togglebutton-selected-color: @agate-button-selected-color;
 @agate-togglebutton-selected-bg-color:  @agate-button-selected-bg-color;
-
-// Scroller
-// ---------------------------------------
-@agate-scroller-scrollbar-thumb-bg-color: #373737;
```

Comment:
Maybe use the same color as the arrow icons. Currently it is @agate-darker-gray, but you can change it to the lighter text-color that you chose in #564. (@agate-gray)